lightning/src/ln/channel.rs

```diff
@@ -11152,6 +11152,11 @@ where
 			})?;
 		debug_assert!(interactive_tx_constructor.take_initiator_first_message().is_none());
 
+		// TODO(splicing): if post_quiescence_action is set, integrate what the user wants to do
+		// into the counterparty-initiated splice. For always-on nodes this probably isn't a useful
+		// optimization, but for often-offline nodes it may be, as we may connect and immediately
+		// go into splicing from both sides.
```

Comment:
Would this happen when both counterparties initiated quiescence simultaneously? Ties should go to the channel opener. Would we lose the user-initiated `quiescent_action` in that case? Or rather would we possibly be stuck when the user tries to initiate the splice again since `quiescent_action` is `Some`?